Internal Server Error: /portal/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 208, in render
    return self._render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 217, in render
    nodelist.append(node.render_annotated(context))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 159, in render
    values = self.sequence.resolve(context, True)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 739, in resolve
    new_obj = func(obj, *arg_vals)
  File "/etc/side_projects/RCBuilder/rcapp/templatetags/fk_filters.py", line 16, in with_ref_to_impact
    return impact_submodels.filter(impact_ref=current_impact_ref)
AttributeError: 'NoneType' object has no attribute 'filter'
[11/Dec/2016 17:55:46] "GET /portal/1 HTTP/1.1" 500 177351
Internal Server Error: /portal/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 208, in render
    return self._render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 217, in render
    nodelist.append(node.render_annotated(context))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 159, in render
    values = self.sequence.resolve(context, True)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 739, in resolve
    new_obj = func(obj, *arg_vals)
  File "/etc/side_projects/RCBuilder/rcapp/templatetags/fk_filters.py", line 16, in with_ref_to_impact
    return impact_submodels.filter(impact_ref=current_impact_ref)
AttributeError: 'NoneType' object has no attribute 'filter'
[11/Dec/2016 17:55:48] "GET /portal/1 HTTP/1.1" 500 177351
Internal Server Error: /portal/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 208, in render
    return self._render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 323, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 217, in render
    nodelist.append(node.render_annotated(context))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 159, in render
    values = self.sequence.resolve(context, True)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 739, in resolve
    new_obj = func(obj, *arg_vals)
  File "/etc/side_projects/RCBuilder/rcapp/templatetags/fk_filters.py", line 16, in with_ref_to_impact
    return impact_submodels.filter(impact_ref=current_impact_ref)
AttributeError: 'NoneType' object has no attribute 'filter'
[11/Dec/2016 17:55:51] "GET /portal/1 HTTP/1.1" 500 177351
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 17:55:44
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Internal Server Error: /portal/1
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
KeyError: 'else'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 84, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/usr/local/lib/python3.4/dist-packages/django/template/response.py", line 66, in resolve_template
    return select_template(template, using=self.using)
  File "/usr/local/lib/python3.4/dist-packages/django/template/loader.py", line 48, in select_template
    return engine.get_template(template_name)
  File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py", line 39, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/usr/local/lib/python3.4/dist-packages/django/template/engine.py", line 160, in get_template
    template, origin = self.find_template(template_name)
  File "/usr/local/lib/python3.4/dist-packages/django/template/engine.py", line 134, in find_template
    name, template_dirs=dirs, skip=skip,
  File "/usr/local/lib/python3.4/dist-packages/django/template/loaders/base.py", line 44, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 191, in __init__
    self.nodelist = self.compile_nodelist()
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 233, in compile_nodelist
    return parser.parse()
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 953, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 953, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/usr/local/lib/python3.4/dist-packages/django/template/defaulttags.py", line 811, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/usr/local/lib/python3.4/dist-packages/django/template/base.py", line 568, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 292: 'else', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
[11/Dec/2016 17:56:20] "GET /portal/1 HTTP/1.1" 500 187337
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 17:56:19
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 17:56:25
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 17:57:21] "GET /portal/1 HTTP/1.1" 200 12891
[11/Dec/2016 17:57:22] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 17:57:22] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 17:57:22] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 17:57:22] "GET /static/css/main.css HTTP/1.1" 200 8957
[11/Dec/2016 17:57:22] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 17:57:22] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 17:57:22] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 17:57:22] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 17:57:22] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 17:57:22] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 17:57:22] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 17:57:22] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 17:57:22] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 17:57:22] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 17:57:22] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 17:57:22] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 17:57:21
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 17:57:27
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:03:22
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:03:23
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:03:27
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:04:07] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:04:07] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:04:07] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:04:07] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:04:07] "GET /static/css/main.css HTTP/1.1" 200 9015
[11/Dec/2016 18:04:07] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:04:07] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:04:07] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:04:07] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:04:07] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:04:07] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:04:07] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:04:07] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:04:07] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:04:07] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:04:08] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:04:08] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:06
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:12
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:04:20] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:04:20] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:04:20] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:04:20] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:04:20] "GET /static/css/main.css HTTP/1.1" 200 9015
[11/Dec/2016 18:04:20] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:04:20] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:04:20] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:04:20] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:04:20] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:04:20] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:04:20] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:04:20] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:04:20] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:04:20] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:04:20] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:04:20] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:19
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:24
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:04:36] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:04:36] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:04:36] "GET /static/css/main.css HTTP/1.1" 200 9018
[11/Dec/2016 18:04:36] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:04:36] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:04:36] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:04:36] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:04:36] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:04:36] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:04:36] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:04:36] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:04:36] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:04:36] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:04:36] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:04:36] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:04:36] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:04:36] "GET /static/img/logo.png HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:04:38] "GET /static/css/font-awesome.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /static/css/owl.carousel.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /static/css/main.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /static/css/prettyPhoto.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:38] "GET /static/css/animate.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:39] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:04:39] "GET /static/css/responsive.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:39] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:04:39] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:04:39] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:04:39] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:04:39] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:04:39] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:04:39] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:04:39] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:36
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:04:52] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:04:52] "GET /static/css/font-awesome.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/prettyPhoto.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/main.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/owl.carousel.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/animate.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/css/responsive.css HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/plugins.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/wow.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/bootstrap.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/owl.carousel.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/main.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/js/prettyPhoto.js HTTP/1.1" 304 0
[11/Dec/2016 18:04:52] "GET /static/img/logo.png HTTP/1.1" 304 0
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:04:41
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:05:47] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:05:47] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:05:47] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:05:47] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:05:47] "GET /static/css/main.css HTTP/1.1" 200 9018
[11/Dec/2016 18:05:47] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:05:47] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:05:47] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/bootstrap.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/prettyPhoto.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/plugins.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/wow.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:05:47] "GET /static/js/owl.carousel.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/js/main.js HTTP/1.1" 304 0
[11/Dec/2016 18:05:47] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:05:45
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:05:51
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:06:15] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:06:15] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:06:15] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:06:15] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:06:15] "GET /static/css/main.css HTTP/1.1" 200 9041
[11/Dec/2016 18:06:15] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:06:15] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:06:15] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:06:15] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:06:15] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:06:15] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:06:15] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:06:15] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:06:15] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:06:15] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:06:15] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:06:15] "GET /static/img/logo.png HTTP/1.1" 304 0
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:06:15
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:06:21
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:06:30] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:06:30] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:06:30] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:06:30] "GET /static/css/main.css HTTP/1.1" 200 9043
[11/Dec/2016 18:06:30] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:06:31] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:06:31] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:06:31] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:06:31] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:06:31] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:06:31] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:06:31] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:06:31] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:06:31] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:06:31] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:06:31] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:06:31] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:06:30
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:08:48] "GET /static/css/owl.carousel.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET / HTTP/1.1" 200 12119
[11/Dec/2016 18:08:48] "GET /static/css/responsive.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/prettyPhoto.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:08:48] "GET /static/css/animate.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:08:48] "GET /static/img/logo.png HTTP/1.1" 200 18815
[11/Dec/2016 18:08:48] "GET /static/img/app-preview-empty.png HTTP/1.1" 200 43649
[11/Dec/2016 18:08:48] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:08:48] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:08:48] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:08:48] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:08:48] "GET /static/css/owl.carousel.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/responsive.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/animate.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/prettyPhoto.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/font-awesome.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/css/main.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:08:48] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:08:48] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:08:48] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:08:48] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:08:48] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:08:48] "GET /static/css/main.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:48] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:08:48] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:08:49] "GET /static/css/font-awesome.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:08:49] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:08:49] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:08:49] "GET /static/img/logo.png HTTP/1.1" 200 18815
[11/Dec/2016 18:08:49] "GET /static/img/header-bg.jpg HTTP/1.1" 200 2383551
[11/Dec/2016 18:08:58] "GET /portal/ HTTP/1.1" 200 10528
[11/Dec/2016 18:09:00] "GET /portal/ HTTP/1.1" 200 10528
[11/Dec/2016 18:09:04] "GET /portal/1 HTTP/1.1" 200 13297
[11/Dec/2016 18:09:04] "GET /static/css/bootstrap.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/font-awesome.min.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/animate.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/prettyPhoto.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/owl.carousel.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/main.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/css/responsive.css HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/plugins.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/bootstrap.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/wow.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/prettyPhoto.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/owl.carousel.min.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/js/main.js HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /static/img/logo.png HTTP/1.1" 304 0
[11/Dec/2016 18:09:04] "GET /portal/1 HTTP/1.1" 200 13297
Not Found: /http:/testp3.pospr.waw.pl/testproxy.php
[11/Dec/2016 18:10:49] "GET http://testp3.pospr.waw.pl/testproxy.php HTTP/1.1" 404 1721
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:06:36
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
[11/Dec/2016 18:11:47] "GET /portal/ HTTP/1.1" 200 10528
[11/Dec/2016 18:11:47] "GET /static/css/font-awesome.min.css HTTP/1.1" 200 20766
[11/Dec/2016 18:11:47] "GET /static/css/main.css HTTP/1.1" 200 9043
[11/Dec/2016 18:11:47] "GET /static/css/owl.carousel.css HTTP/1.1" 200 3149
[11/Dec/2016 18:11:47] "GET /static/css/prettyPhoto.css HTTP/1.1" 200 19930
[11/Dec/2016 18:11:47] "GET /static/css/responsive.css HTTP/1.1" 200 1013
[11/Dec/2016 18:11:47] "GET /static/js/plugins.js HTTP/1.1" 200 733
[11/Dec/2016 18:11:47] "GET /static/css/animate.css HTTP/1.1" 200 59839
[11/Dec/2016 18:11:47] "GET /static/js/vendor/modernizr-2.6.2.min.js HTTP/1.1" 200 15414
[11/Dec/2016 18:11:47] "GET /static/css/bootstrap.min.css HTTP/1.1" 200 116743
[11/Dec/2016 18:11:47] "GET /static/js/wow.min.js HTTP/1.1" 200 7083
[11/Dec/2016 18:11:47] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 31819
[11/Dec/2016 18:11:47] "GET /static/js/prettyPhoto.js HTTP/1.1" 200 22060
[11/Dec/2016 18:11:47] "GET /static/js/owl.carousel.min.js HTTP/1.1" 200 23890
[11/Dec/2016 18:11:47] "GET /static/js/main.js HTTP/1.1" 200 1554
[11/Dec/2016 18:11:47] "GET /static/js/vendor/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[11/Dec/2016 18:11:47] "GET /static/img/logo.png HTTP/1.1" 200 18815
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:11:45
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Not Found: /portal/create/
[11/Dec/2016 18:11:54] "GET /portal/create/ HTTP/1.1" 404 2684
Performing system checks...

System check identified no issues (0 silenced).
December 11, 2016 - 18:11:50
Django version 1.10, using settings 'RCBuilder.settings'
Starting development server at http://0.0.0.0:8080/
Quit the server with CONTROL-C.
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f60c0420950>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/commands/runserver.py", line 121, in inner_run
    self.check(display_num_errors=True)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 385, in check
    include_deployment_checks=include_deployment_checks,
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 372, in _run_checks
    return checks.run_checks(**kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 24, in check_resolver
    for pattern in resolver.url_patterns:
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 310, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 303, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/RCBuilder/urls.py", line 25, in <module>
    url(r'^portal/', include('rcapp.urls')),
  File "/usr/local/lib/python3.4/dist-packages/django/conf/urls/__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/rcapp/urls.py", line 8, in <module>
    url(r'^create/', view.RCCreateView.as_view(), name='rccreateview'),
NameError: name 'view' is not defined
Performing system checks...

Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f35da03cb70>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/commands/runserver.py", line 121, in inner_run
    self.check(display_num_errors=True)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 385, in check
    include_deployment_checks=include_deployment_checks,
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 372, in _run_checks
    return checks.run_checks(**kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 24, in check_resolver
    for pattern in resolver.url_patterns:
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 310, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 303, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/RCBuilder/urls.py", line 25, in <module>
    url(r'^portal/', include('rcapp.urls')),
  File "/usr/local/lib/python3.4/dist-packages/django/conf/urls/__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/rcapp/urls.py", line 8, in <module>
    url(r'^create/', view.RCCreateView.as_view(), name='rccreateview'),
NameError: name 'view' is not defined
Performing system checks...

Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7ff60db74950>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/commands/runserver.py", line 121, in inner_run
    self.check(display_num_errors=True)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 385, in check
    include_deployment_checks=include_deployment_checks,
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 372, in _run_checks
    return checks.run_checks(**kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 24, in check_resolver
    for pattern in resolver.url_patterns:
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 310, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 303, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/RCBuilder/urls.py", line 25, in <module>
    url(r'^portal/', include('rcapp.urls')),
  File "/usr/local/lib/python3.4/dist-packages/django/conf/urls/__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/rcapp/urls.py", line 8, in <module>
    url(r'^create/', view.RCCreateView.as_view(), name='rccreateview'),
NameError: name 'view' is not defined
Performing system checks...

Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7fdfc7c18b70>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/commands/runserver.py", line 121, in inner_run
    self.check(display_num_errors=True)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 385, in check
    include_deployment_checks=include_deployment_checks,
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/base.py", line 372, in _run_checks
    return checks.run_checks(**kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/registry.py", line 81, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/usr/local/lib/python3.4/dist-packages/django/core/checks/urls.py", line 24, in check_resolver
    for pattern in resolver.url_patterns:
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 310, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/usr/local/lib/python3.4/dist-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/local/lib/python3.4/dist-packages/django/urls/resolvers.py", line 303, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/RCBuilder/urls.py", line 25, in <module>
    url(r'^portal/', include('rcapp.urls')),
  File "/usr/local/lib/python3.4/dist-packages/django/conf/urls/__init__.py", line 50, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/etc/side_projects/RCBuilder/rcapp/urls.py", line 8, in <module>
    url(r'^create/', view.RCCreateView.as_view(), name='rccreateview'),
NameError: name 'view' is not defined
Performing system checks...

